src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -62,7 +70,9 @@ function useChatUtilsHook(): UseChatUtilsHook {
       }
     }
 
-    refresh('channelList');
+    refresh('channelInfo');
+    refresh('chat');
+    refresh('user');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The refresh function now targets 'channelInfo', 'chat', and 'user' instead of 'channelList'. This change seems to be more specific and targeted, which could potentially improve performance by reducing unnecessary data fetches or updates.
```diff
-   refresh('channelList');
+   refresh('channelInfo');
+   refresh('chat');
+   refresh('user');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->